src/components/NewCommentForm.tsx

```diff
@@ -1,8 +1,132 @@
-import React from 'react';
+import classNames from 'classnames';
+import React, { useCallback, useState } from 'react';
+import { IComment } from '../types/Comment';
+import { ErrorTypes } from '../types/ErrorTypes';
+import { IPost } from '../types/Post';
+import { createComment } from '../utils/fetchClient';
+
+interface Props {
+  post: IPost;
+  handleAddComment: (comment: IComment) => void;
+}
+
+interface Errors {
+  isErrorName: ErrorTypes | null;
+  isErrorEmail: ErrorTypes | null;
+  isErrorComment: ErrorTypes | null;
```

Comment:
Prefixes is\are\can etc. are used only in boolean values.
```suggestion
  ErrorName: ErrorTypes | null;
  ErrorEmail: ErrorTypes | null;
  ErrorComment: ErrorTypes | null;
```